firewood/src/storage/buffer.rs

```diff
@@ -342,43 +334,31 @@ async fn init_wal(
 
 async fn run_wal_queue(
```

Comment:
Similar question to above -- what does this do now that we're removing undo/redo persistence?

Reply:
Same as above, it is actually persisting the dirty pages.